recipe detail ingredients  customization (#29)
miamtech/mealz-documentation: docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailIngredients.mdx

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ### ingredients
> 
> To create your own recipe detail Iingredients template you have to create class that implement `Ingredients`

Reviewer: ```suggestion
### Ingredients

To create your own recipe detail ingredients template you create a class that implements `Ingredients`
```
